packages/nitro-protocol/test/contracts/ninja-nitro/SingleChannelAdjudicator/claim.test.ts

```diff
@@ -0,0 +1,337 @@
+      addresses.g = guarantor.id;
+      // Transform input data (unpack addresses and BigNumber amounts)
+      [heldBefore, tOutcomeBefore, tOutcomeAfter, heldAfter, payouts] = [
+        heldBefore,
+        tOutcomeBefore,
+        tOutcomeAfter,
+        heldAfter,
+        payouts,
+      ].map(object => replaceAddressesAndBigNumberify(object, addresses) as AssetOutcomeShortHand);
+      guaranteeDestinations = guaranteeDestinations.map(x => addresses[x]);
+      // Fund the guarantor channel
+      let gasUsed;
+      gasUsed = await guarantor.depositETH(heldBefore[guarantor.id]);
+      await writeGasConsumption(
+        'SingleChannelAdjudicator.claim.gas.md',
+        name + ': send ETH to Guarantor',
+        gasUsed
+      );
+      // DEPLOY GUARANTOR CHANNEL
+      gasUsed = await guarantor.deploy();
+      await writeGasConsumption(
+        'SingleChannelAdjudicator.claim.gas.md',
+        name + ': deploy Guarantor',
+        gasUsed
+      );
+      // Compute an appropriate guarantee for the guarantor (using only ETH)
+      const guarantee = {
+        destinations: guaranteeDestinations,
+        targetChannelId: target.id,
+      };
+      const guarantorOutcome: Outcome = [{assetHolderAddress: constants.AddressZero, guarantee}];
+      if (guaranteeDestinations.length > 0) {
+        // CONCLUDE GUARANTOR CHANNEL
+        gasUsed = await guarantor.conclude(guarantorOutcome);
+        await writeGasConsumption(
+          'SingleChannelAdjudicator.claim.gas.md',
+          name + ': conclude Guarantor',
+          gasUsed
+        );
+      } else guarantor.outcome = guarantorOutcome; // Set this so that the claim tx should revert in the way we expect
+      // Compute an appropriate allocation outcome for the target (using only ETH)
+      const allocation = [];
+      Object.keys(tOutcomeBefore).forEach(key =>
+        allocation.push({destination: key, amount: tOutcomeBefore[key]})
+      );
+      const targetOutcome: Outcome = [
+        {assetHolderAddress: constants.AddressZero, allocationItems: allocation},
+      ];
+
+      // DEPLOY TARGET CHANNEL
+      gasUsed = await target.deploy();
+      await writeGasConsumption(
+        'SingleChannelAdjudicator.claim.gas.md',
+        name + ': deploy Target',
+        gasUsed
+      );
+      // CONCLUDE TARGET CHANNEL
+      if (Object.keys(tOutcomeBefore).length > 0) {
+        gasUsed = await target.conclude(targetOutcome);
+        await writeGasConsumption(
+          'SingleChannelAdjudicator.claim.gas.md',
+          name + ': conclude Target',
+          gasUsed
+        );
+      }
+
+      const tx = target.claimTx(guarantor, indices);
+      // Call method in a slightly different way if expecting a revert
+      if (reason) {
+        await expectRevert(() => tx, reason);
+      } else {
+        const balancesBefore = await getBalances(payouts);
+        // Extract logs
+        const {events: eventsFromTx, gasUsed} = await (await tx).wait();
```

Comment:
Removing unused variable.

```suggestion
        const {gasUsed} = await (await tx).wait();
```